code/modules/mob/living/living.dm

```diff
@@ -845,11 +845,75 @@ default behaviour is:
 	..()
 
 //damage/heal the mob ears and adjust the deaf amount
-/mob/living/adjustEarDamage(var/damage, var/deaf)
+/mob/living/adjustEarDamage(var/damage, var/deaf, var/ringing = FALSE)
+	var/alreadydeaf = FALSE
+	if (ear_deaf)
+		alreadydeaf = TRUE
+
+	if (ishuman(src) && damage > 0)
+		var/mob/living/carbon/human/H = src
+		if (H.is_hearing_sensitive())
+			if (H.is_listening()) // if the person is listening in, the effect is way worse
+				if (H.is_hearing_sensitive() == 2)
+					damage *= 2
+				else
+					damage = round(damage *= 1.5, 1)
+				H.stop_listening()
+			else
+				if (H.is_hearing_sensitive() == 2)
+					damage = round(damage *= 1.4, 1)
+				else
+					damage = round(damage *= 1.2, 1)
+
 	ear_damage = max(0, ear_damage + damage)
 	ear_deaf = max(0, ear_deaf + deaf)
 
+	if (ringing && !alreadydeaf)
+		if (ear_damage >= 5)
+			if (ear_damage >= 15)
+				to_chat(src, SPAN_DANGER("Your ears start to ring badly!"))
+			else
+				to_chat(src, SPAN_DANGER("Your ears start to ring!"))
+
+
+// Intensity 1: mild, 2: hurts, 3: very painful, 4: extremely painful, 5: that's going to leave some damage
+// Sensitive_only: If yes, only those with sensitive hearing are affected
+// Listening_pain: Increases the intensity by the listed amount if the person is listening in
+/mob/living/carbon/proc/earpain(var/intensity, var/sensitive_only = FALSE, var/listening_pain = 0) 
+	if (ear_deaf)
+		return
+	if (ishuman(src))
```

Comment:
make a /mob/living/carbon/human subtype of the proc instead of checking if living is a human